docs/Operation-Architecture/Zenko_CLI/Zenko_from_the_Command_Line.rst

```diff
@@ -1,27 +1,44 @@
 Zenko from the Command Line
 ===========================
 
+Zenko supports command-line interactions for a limited set of Amazon
+S3 API calls and to access its own Backbeat server.
+
+Enabling command-line interactions enables programmatic access to
+the following features:
+
+.. toctree::
+   :maxdepth: 1
+
+   CRR Metrics and Healthcheck <CRR_Metrics_and_Health>
+   CRR Retry <CRR_Retry>
+   CRR Pause and Resume <CRR_Pause_&_Resume>
+   CRR Statistics<CRR_statistics>
+   Object Lifecycle Management <../Lifecycle_Management/Object_Lifecycle_Management>
+
+Accessing Zenko from the command line requires the following setup tasks.
+   
 S3 API
 ------
 
-Zenko supports a limited set of S3 API commands. For a comprehensive
+Zenko supports a limited set of S3 API commands. For a comprehensive
 listing of supported S3 commands, see the *Scality Zenko Enterprise Reference
```

Comment:
Should we mention the *enterprise* manual in Zenko public docs?

Reply:
The Reference Manual has yet to be converted and is only available in Enterprise form (from Flare source).